inspirehep/modules/disambiguation/config.py

```diff
@@ -25,16 +25,19 @@
 from __future__ import absolute_import, division, print_function
 
 
-DISAMBIGUATION_CLUSTERS_PATH = '/tmp/clusters.json'
+DISAMBIGUATION_BASE_PATH = '/var/data/disambiguation/'
+"""Base path to all files used in this module."""
+
+DISAMBIGUATION_CLUSTERS_PATH = DISAMBIGUATION_BASE_PATH + 'clusters.json'
```

Comment:
Trivial but, we might want to save the predicted clusters later as well. So we might want to rename this to DISAMBIGUATION_INPUT_CLUSTERS_PATH.

Reply:
I don't disagree with this comment, but let's wait to do it until we absolutely need it. Please remind me if I forget!